Framework/Backend/http/server.js

```diff
@@ -417,7 +418,7 @@ class HttpServer {
         personid: details.cern_person_id,
         name: details.name,
         access: this.authorise(details),
-        token: this.jwt.generateToken(details.cern_person_id, details.cern_upn, this.authorise(details)),
+        token: this.jwt.generateToken(details.cern_person_id, details.cern_upn, details.name, this.authorise(details)),
```

Comment:
For compatiblity reasons, wouldn't it be better to add `name` as parameter at the end of the list of existing parameters so that any forgotten uses are covered? Preferably with a default value?

For example we use the `generateToken` method in `server.js` as well which I do not see being updated here

Reply:
I see your point but there's already one optional argument in that method, in addition it's difficult to have default value for name.

Regarding `generateToken` method in `server.js`, could you clarify? It looks good to me.

Reply:
I do not think there is anything preventing us from setting default values to parameters in method declaration. 

1. While for the `server.js` I am referring to this particular case: https://github.com/AliceO2Group/WebUi/blob/dev/Framework/Backend/http/server.js#L420 

In here, now the method expects the 3rd parameter to be the name but the server will send the authorise code.

2. While in here: https://github.com/AliceO2Group/WebUi/blob/dev/Framework/Backend/http/server.js#L240
We pass a name for the username and in the future I guess we will want to display the name of the user, so I think this line should be updated as well in this PR